birdhouse/.gitignore

```diff
@@ -0,0 +1 @@
+./config
```

Comment:
Since we are purposely moving components, should we consider leaving this un-ignored? 
Could help detect an invalid definition that refers to the old path.

Reply:
I think we have to ignore it so that the autoupdate code works. Right @tlvu ?

Reply:
Since it would be a major update that could remove components if misconfigured, I guess it that it would feel safer to purposely break the autoupdate to have node admins make sure they set up their `EXTRA_CONF_DIRS` properly and clean up leftover configs if applicable before updating. However, if others don't feel this could be an issue, I'm fine with having the gitignore also. I'm not personally using autoupdate.

Reply:
I'm not either so let's let @tlvu decide this one

Reply:
I'd rather prefer not breaking the autodeploy.  We had already the plan to do this: add the new paths in advance to `env.local` so it is already forward compatible when this PR merge so no manual interventions required.

However I would also ping PCIC in this PR so they are aware of this major change.